modules/private/hook.go

```diff
@@ -86,7 +85,6 @@ type HookProcReceiveRefResult struct {
 func HookPreReceive(ctx context.Context, ownerName, repoName string, opts HookOptions) ResponseExtra {
 	reqURL := setting.LocalURL + fmt.Sprintf("api/internal/hook/pre-receive/%s/%s", url.PathEscape(ownerName), url.PathEscape(repoName))
 	req := newInternalRequestAPI(ctx, reqURL, "POST", opts)
-	req.SetReadWriteTimeout(time.Duration(60+len(opts.OldCommitIDs)) * time.Second)
```

Comment:
But we can't only delete this line ....


![image](https://github.com/user-attachments/assets/9a3636eb-e772-4d95-b4d2-544093609b98)

Reply:
Well... Thanks for noting.